examples/react-native/package.json

```diff
@@ -6,8 +6,8 @@
     "test": "jest"
   },
   "dependencies": {
-    "react": "15.4.2",
-    "react-native": "^0.40.0"
+    "react": "16.2.0",
```

Comment:
I get a peer dep warning when installing, is that ok?

`warning " > react-native@0.51.0" has incorrect peer dependency "react@16.0.0".`

Reply:
It works, so I'm fine with it.